innersource-activity-score.md

```diff
@@ -0,0 +1,92 @@
+## Title 
+InnerSource Activity Score
+
+## Patlet
+The InnerSource activity score is a numeric value that represents the (GitHub) activity of an InnerSource project. It can be used to create a ranked list of projects for the [InnerSource portal](innersource-portal.md) so that potential contributors can easily find the most active projects. It is derived automatically from GitHub repository statistics and may be enriched with KPIs from other tools or manual evaluations. 
```

Comment:
From the template:
> Concise 1-2 sentence description of the problem and solution. Readers may quickly review dozens of these patlets to discover and browse the larger library of patterns. 

Squeezing everything into two sentences is really tough but still worth trying for the benefit of the readers :) Even just stating the problem first, and then following with the solution can already provide a lot of value.

Let me try:

```suggestion
Potential contributors want to find active InnerSource projects in need of their help. By calculating an InnerSource activity score for each project, a ranked list of projects can be created (e.g. on the [InnerSource portal](innersource-portal.md)), so that potential contributors can more easily determine which project they want to contribute to.
```